scripts/symlink_artifacts.sh

```diff
@@ -21,7 +21,7 @@ symlink_artifact() {
     local root_path=$3
     local skip_sanity_check=${4:-"0"}
 
-    local artifact_src_path="$(realpath "${src_path}/${binary_path}")"
```

Comment:
Instead of removing `realpath` I would just make it `realpath -m`.

Reply:
ok